src/server/controllers/LoginController.ts

```diff
@@ -44,7 +44,7 @@ export class LoginController implements LoginControllerInterface {
     const { email }: EmailProperty = req.body
 
     try {
-      await this.authService.generateOtp(email)
+      await this.authService.generateOtp(email, req.ip)
```

Comment:
this will be the cloudflare proxy's IP unless you read the CF-Connecting-IP header